suse/distributionscanner.go

```diff
@@ -3,13 +3,16 @@ package suse
 import (
 	"bytes"
 	"context"
-	"regexp"
+	"fmt"
 	"runtime/trace"
 
+	"github.com/Masterminds/semver"
 	"github.com/quay/zlog"
 
 	"github.com/quay/claircore"
 	"github.com/quay/claircore/indexer"
+	"github.com/quay/claircore/osrelease"
+	"github.com/quay/claircore/pkg/cpe"
 )
 
 // Suse Enterprise Server has service pack releases however their security database files are bundled together
```

Comment:
also just realized this: just checking: no need to update the version, right? I don't think anything noticeable to end users changed here

Reply:
yeah, the indexing should be the same I believe